.gitignore

```diff
@@ -56,3 +56,5 @@ typings/
 
 # dotenv environment variables file
 .env
+
+data/
```

Comment:
[gitignore.io](https://gitignore.io) fyi

Reply:
@dwalintukan yes. It is a mount vol I use to store db data and qtum data

Reply:
👍 ya that website is handy